packages/google_mobile_ads/lib/src/ad_listeners.dart

```diff
@@ -27,7 +27,7 @@ typedef FullScreenAdLoadErrorCallback = void Function(LoadAdError error);
 
 /// The callback type for when a user earns a reward from a [RewardedAd].
```

Comment:
```suggestion
/// The callback type for when a user earns a reward.
```

Matches https://developers.google.com/android/reference/com/google/android/gms/ads/OnUserEarnedRewardListener